.gitignore

```diff
@@ -0,0 +1,2 @@
+output/
+docker-compose.yaml
```

Comment:
This doesn't exist anymore

Reply:
it does ;) in my local git repo on my machine and I want to be sure never to commit it (that's the point)